docs/content/en/docs/user-guide/managing-piped/configuration-reference.md

```diff
@@ -255,7 +255,9 @@ Must be one of the following structs:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
-| hookURL | string | The hookURL of a slack channel. | Yes |
+| hookURL | string | The hookURL of a slack channel. | No |
+| oauthToken | string | The token of a slack api. | No |
```

Comment:
```suggestion
| oauthToken | string | The token for slack api use. | No |
```
Would be best if you can also add a link to how to get slack oauth token docs.

Reply:
I updated it.